esp/esp/program/models/__init__.py

```diff
@@ -50,19 +50,27 @@
 from django.db.models import Q
 from django.db.models.query import QuerySet
 
+from esp.users.models import ESPUser, Permission
 from esp.cache import cache_function
 from esp.cache.key_set import wildcard
 from esp.cal.models import Event
 from esp.customforms.linkfields import CustomFormsLinkModel
 from esp.datatree.models import *
 from esp.db.fields import AjaxForeignKey
-from esp.middleware import ESPError, AjaxError
+from esp.middleware import ESPError, AjaxError, ESPError_Log
 from esp.tagdict.models import Tag
 from esp.users.models import ContactInfo, StudentInfo, TeacherInfo, EducatorInfo, GuardianInfo, ESPUser, shirt_sizes, shirt_types, Record
 from esp.utils.expirable_model import ExpirableModel
 from esp.utils.formats import format_lazy
 from esp.qsdmedia.models import Media
 
+PROFILE_MAX_AGE_DAYS = 5
+
+class ProgramAccessError(ESPError_Log):
+        def __init__(self, msg):
+            self.msg = msg
+            super(ProfileNotFoundException, self).__init__(msg)
```

Comment:
This is mismatched with the class name ProgramAccessError.  Also, I think the tab indentation is off.